### PR TITLE
fix(dockerfile): Healthcheck parameters

### PR DESCRIPTION
**Pull request by reggermont**
## Current behaviour

**TL;DR:** Container is taking 5 minutes before being healthy

Docker is checking health of the container 0 second after the start ([default value](https://docs.docker.com/engine/reference/builder/#healthcheck) and then every 5 minutes, so 5 minutes later and 10 minutes later. 

After 0 second, the service is, of course, not ready. So the healthcheck realisticly waits for 5 minutes before trying to check if the container is healthy.

This affects traefik reverse-proxy based servers, as it relates on the healthcheck if it exists, meaning that ferdi server takes 5 minutes before being available

## New behaviour

**TL;DR:** Container is now taking 5 seconds before being healthy for 99% of setups, and more for the 1% left

(The values are a proposal, may be changed)

Now Docker waits for 5 seconds before checking container's health, which should be enough for 99% of the setups, and will retry after 30 seconds 4 more times if health check fails (with a maximum of 2 minutes and 5 seconds, which should be enough for the 1% left)

## How to test

You can try with `docker run eggermont/ferdi-server:2021-12-19` ([link to Docker Hub](https://hub.docker.com/r/eggermont/ferdi-server)) and making a `docker ps` to see container's health.

### CODE REVIEW

**Comment by reggermont** [comment]
Proof of concept on a small server:

```
CONTAINER ID   IMAGE                               COMMAND                  CREATED              STATUS                        PORTS                                                                                                                 NAMES
a5d9e4e320d9   eggermont/ferdi-server:2021-12-19   "docker-entrypoint.s…"   About a minute ago   Up About a minute (healthy)
```

**Comment by kytwb** [comment]
LGTM, thanks.

**Comment by kytwb** [comment]
@all-contributors please add @reggermont for code
